### PR TITLE
Integrate python sdk to fetch forta alerts using graphql api

### PR DESCRIPTION
**Pull request by Robert-H-Leonard**
- [x] Added python sdk support to fetch bot alerts using forta graphql api
- [x] Tested locally with a script
- [x] Validate all query filters and response object serialization

Screen Shot 2022-07-11 at 11 12 19 AM

### CODE REVIEW

File: python-sdk/src/forta_agent/utils.py [hunk 1]
```diff
@@ -81,6 +82,24 @@ def create_transaction_event(dict):
...
+    headers = {"content-type": "application/json"}
+    query_options = AlertQueryOptions(dict)
+    payload = query_options.get_query()
+    response = requests.request("POST", forta_api, json=payload, headers=headers)
+
+    if response.status_code == 200:
+        data = response.json().get('data')
+
+        if data:
+            return data.get('alerts')
```
Comment (haseebrabbani) [inline python-sdk/src/forta_agent/utils.py:97]: will the paging info also be returned? (or just alerts?)

Comment (Robert-H-Leonard) [reply python-sdk/src/forta_agent/utils.py:97]: Yep the page info is returned. The returned object looks like this:

`{
 alerts: [...],
 pageInfo: {..}
}`

In the image in the PR description you can see what the pageInfo looks like in the response.

File: python-sdk/src/forta_agent/forta_graphql.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+              }
+            }
+          }
+          """
+        query_variables = vars(self)
+        filtered_query_variables = {k:v for k,v in query_variables.items() if v is not None}
+        return dict(query=query, variables={"input": filtered_query_variables})
+
+
+class AlertsResponse:
```
[the remark this answers is not in the document]
Comment (Robert-H-Leonard) [reply python-sdk/src/forta_agent/forta_graphql.py:70]: Nope, the goal of this is to be a class devs can use to serialize the response they get when they fetch alerts.

File: python-sdk/src/forta_agent/utils.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Robert-H-Leonard) [reply python-sdk/src/forta_agent/utils.py:97]: The classes inside of `forta_graphql.py` (`AlertsResponse` and `AlertCursor`) are the shapes of the fields returned

File: python-sdk/src/forta_agent/utils.py [hunk 3]
```diff
@@ -2,6 +2,9 @@
 import os
 from jsonc_parser.parser import JsoncParser
 import sha3
+import requests
+
+from forta_agent.forta_graphql import AlertsResponse
```
Comment (haseebrabbani) [inline python-sdk/src/forta_agent/utils.py:7]: we should use a relative import here i.e. `from .forta_graphql import AlertsResponse`

Comment (Robert-H-Leonard) [reply python-sdk/src/forta_agent/utils.py]: Updated

**Pull request merged**